### PR TITLE
Improved styling of scan section in shipment detail page (#220quwn)

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues

Closes # 

### Screenshots of Visual Changes before/after (If There Are Any)
image

localhost_8101_shipment_12163

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [X] I read and followed [contribution rules](https://github.com/hotwax/receiving#contribution-guideline)

### CODE REVIEW

**Review by dt2patel — CHANGES_REQUESTED** [review]
This method is not responsive and will not work great on mobile devices. Please refer to the po detail page styling. I am not sure if that will work on mobile either but see if you are able to use that styling as a foundation to make this responsive.

**Comment by azkyakhan** [comment]
The style used in the PO detail page was not providing a responsive view. I tried to achieve both mobile and desktop views using this styling only by using media queries.

https://user-images.githubusercontent.com/82817616/154202562-05a82235-9987-42da-9c2f-7615eb7d6c9f.mp4

File: src/views/ShipmentDetails.vue [hunk 1]
```diff
@@ -228,4 +231,13 @@ img {
 .product-count {
   min-width: 9ch;
 }
+
+@media (min-width: 636px) {
+  .shipment-scanner {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
```
Comment (dt2patel) [inline src/views/ShipmentDetails.vue:238]: can we do this without a media query if we use auto fit/fill properties of css grid

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/ShipmentDetails.vue [hunk 2]
```diff
@@ -15,17 +15,20 @@
...
-          <ion-label>
-            <ion-input :placeholder="$t('Scan barcodes to receive')"></ion-input>
-          </ion-label>
-          <ion-buttons>
-            <ion-button class="action-button" fill="outline" color="secondary" @click="scanCode()">
+
+        <div class="shipment-scanner">
+          <ion-item>
+            <ion-label>{{ $t("Scan Items") }}</ion-label>
+            <ion-label>
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/views/ShipmentDetails.vue]: Removed ion-label

File: src/views/ShipmentDetails.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/ShipmentDetails.vue]: Used auto-fit property

**Review by dt2patel — APPROVED** [review]

**Pull request merged**